### DISCUSSION TITLE
LCD i2c

### DISCUSSION THREAD

**Discussion by hobbit666** [jfoot/Bus-Departure-Board#37] [Q&A]
https://www.ebay.co.uk/itm/155430133417?var=455632070677
Got one these (20x4) LCD screens.

Was wondering could i mod the project to use one instead of the OLED screens?

**Comment by jfoot** [comment]
Hey @hobbit666, unfortunately I don't think this would work, you'd need a display which is the same resolution as the suggested (256X64). Additionally, while an i2c display should work in theory, with some very minor code changes, I can't make any guarantees as I've known people in the past have issues. So, I'd stick with an SPI display where possible.  Hope that answers your questions – Jonathan.
